x-pack/plugins/fleet/public/applications/fleet/app.tsx

```diff
@@ -253,7 +253,8 @@ export const FleetAppContext: React.FC<{
     theme$,
     fleetStatus,
   }) => {
-    const isDarkMode = useObservable<boolean>(startServices.uiSettings.get$('theme:darkMode'));
+    const darkModeObservable = useObservable<boolean>(startServices.theme.theme$);
```

Comment:
Looks like the types are not correct anymore

```suggestion
    const darkModeObservable = useObservable(startServices.theme.theme$);
```